mitodl/micromasters
# Added possibility to link external program pages

#### What are the relevant tickets?
closes #598 

#### What's this PR do?
allows the CMS users to specify an external link for the program page. The internal program page will not be linked from the homepage

#### Where should the reviewer start?
`cms/models.py`, then `cms/templates/cms/home_page.html`

#### How should this be manually tested?

- Add an URL for the program page in the CMS
- Verify that the program in the homepage links such URL
